hoobs.json

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "hoobs",
+    "release": "11.3-RELEASE",
+    "artifact": "https://github.com/rahul-r/iocage-plugin-hoobs.git",
+    "pkgs": [
+        "python",
+        "perl5",
+        "sudo",
```

Comment:
Is `sudo` really needed? Can't find any usage of it in `post_install.sh`. I've tried to install the plugin without it and that worked good as well.

Reply:
`sudo` is not needed for `post_install.sh` but used by the _HOOBS_ application. Without it, the application will fail to start.

This is the error message without `sudo`
```node
⠹ Enviornment: Clearing NPM Cacheevents.js:293
      throw er; // Unhandled 'error' event
      ^

Error: spawn sudo ENOENT
```